### PR TITLE
urlTemplate.samples{} replaces mds3.variant2sample.url

### PR DESCRIPTION
**Pull request by xzhou82**
## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [ ] Tests: added and/or passed unit and integration tests, or N/A
- [ ] Todos: commented or documented, or N/A
- [ ] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/matrix.interactivity.js [hunk 1]
```diff
@@ -188,9 +191,11 @@ export function setInteractivity(self) {
 					const menuDiv = self.dom.clickMenu.d
 						.append('div')
 						.attr('class', 'sja_menuoption sja_sharp_border')
-						.text(`${s.controlLabels.Sample} summary: ${sampleData._SAMPLENAME_}`)
+						.html(
+							`${s.controlLabels.Sample} summary: <span class='linkText'>${sampleData._SAMPLENAME_} ${svgIcons.externalLink}</span>`
```
Comment (xzhou82) [inline client/plots/matrix.interactivity.js:195]: do not show word summary. it's what gdc calls and not universal
just show `Case: [link]`

File: client/plots/matrix.interactivity.js [hunk 2]
```diff
@@ -188,9 +191,11 @@ export function setInteractivity(self) {
 					const menuDiv = self.dom.clickMenu.d
 						.append('div')
 						.attr('class', 'sja_menuoption sja_sharp_border')
```
Comment (xzhou82) [inline client/plots/matrix.interactivity.js:193]: remove class. this class is for menu option.

Comment (congyu-lu) [reply client/plots/matrix.interactivity.js:193]: this class gives Case/Gene: [link] the same style as the other options ("Disco plot" and "Methylation array")

**Review by siosonel — APPROVED** [review]

File: client/plots/matrix.interactivity.js (hunk 2 above)
Comment (siosonel) [reply client/plots/matrix.interactivity.js:193]: can fix this in a separate branch

**Pull request merged**